disco_aws_automation/disco_storage.py

```diff
@@ -120,15 +121,16 @@ class DiscoStorage(object):
     Wrapper class to handle all DiscoAWS storage functions
     """
 
-    def __init__(self, connection=None):
+    def __init__(self, connection=None, environment_name=None):
         self.connection = connection if connection else boto.connect_ec2()
+        self.environment_name = environment_name
```

Comment:
We usually do that in the `bin` files. For example https://github.com/amplifylitco/asiaq/blob/master/bin/disco_snapshot.py#L88